README.md

```diff
@@ -439,3 +439,30 @@ We recommend querying using payment key hashes (`addr_vkh`) when possible (other
 
 
 </details>
+
+<details>
+  <summary>milkomeda</summary>
+
+  This endpoint is used to get milkomeda info data (minValue, address & tokens list per deployment/network)
+
+  Input
+
+  None (GET request)
+
+  Output
+
+  ```js
+  {
+    minimumValue: number,
+    address: string,
+    tokensPerNetwork: { // proposed networks are the most probable for now
+      internalTestnet: {
+        tokens: [ { name: string, policyId: string }, ]
+      },
+      publicMainnet: {
+        tokens: [ { name: string, policyId: string }, ]
```

Comment:
I suggest you mention the format of this string. I assume it will be hex-encoded bytes?

Reply:
For the `token` payload I'm not sure if this is all info we need tbh. Just added stuff that seemed reasonable. 
I agree, the `policyId` could be specified to be hex-encoded bytes - I'll add that